src/filters/fs.rs

```diff
@@ -258,7 +261,9 @@ impl Reply for File {
     }
 }
 
-fn file_reply(
+/// An internal helper for serving static files. Usually, you'll want to use this with `warp::fs::file`, but if the file's path is based on something extracted with a filter,
```

Comment:
Good practice is to split the first and second sentence with a blank line, and rustdoc will then use the first sentence as the "high level" text description in `docs.rs`, etc, but all of it will be used if you go directly to the docs for the specific item in question.

Reply:
I never knew this, thanks for the tip!